packages/randoml/randoml.0.1.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Generating cryptographically-secure random numbers"
+description:
+  "A library relying on Rust rand to generate random numbers. The random numbers are cryptographically secure."
+maintainer: ["david@o1labs.org"]
+authors: ["David Wong"]
+license: "ISC"
+homepage: "https://github.com/mimoo/randoml"
+bug-reports: "https://github.com/mimoo/randoml/issues"
+depends: [
+  "bignum" {>= "v0.14.0"}
+  "dune" {>= "2.8" & >= "2.8.0"}
```

Comment:
```suggestion
  "dune" {>= "2.8.0"}
```

Reply:
[opam-dune-lint](https://opam.ocaml.org/packages/opam-dune-lint/) does weird things o.o

Reply:
Don’t worry about that. It is a know bug. You can avoid it by removing dune from the dependencies in dune-project. Dune automatically adds itself to the list in any case